index.js

```diff
@@ -20,46 +20,54 @@ app.listen(3000, function(req, res){
 });
```

Comment:
@Firnaa5 You can use the `var PORT`  from `L4` instead of hard-coding `3000` port here.